README.md

```diff
@@ -68,6 +68,31 @@ metadata:
 type: Opaque
 ```
 
+Your Exoscale Key should have at least the following `operation` for your domain: `list-dns-domain-records`, `get-dns-domain-record`, `list-dns-domains`, `update-dns-domain-record`, `create-dns-domain-record`, `delete-dns-domain-record`
```

Comment:
```suggestion
The IAM role policy of your key should allow at least the following `operation`s for your domain: `list-dns-domain-records`, `get-dns-domain-record`, `list-dns-domains`, `update-dns-domain-record`, `create-dns-domain-record`, `delete-dns-domain-record`
```

Reply:
There is already a note about required permissions [here](https://github.com/exoscale/cert-manager-webhook-exoscale/blob/master/README.md?plain=1#L16).

We can remove the old note now, it referenced the old IAM key type anyway.